dashboard/app/controllers/application_controller.rb

```diff
@@ -115,6 +115,9 @@ def milestone_response(options)
     script_level = options[:script_level]
 
     if script_level
+      response[:script_id] = script_level.script.id
+      response[:level_id] = script_level.level.id
```

Comment:
these are placed here rather than within the `HintViewRequest.enabled?` block because I'm going to be using them for the Fun-O-Meter, and they'll need to be moved here this sprint anyway